launch_local.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
+
+set -e
+set -u
+
+# Creating conf.env file
+pushd ./docker/conf/env || exit
+./create-env-linux-1.sh
+popd || exit
+
+# Clean the swarm
+docker swarm leave --force
+docker ps -a
+echo
+docker network prune -f
+docker system prune --volumes -f
+docker network ls
+echo
```

Comment:
We should probably warn before doing this.

Reply:
Added a confirmation step for this one.